cpp/src/arrow/flight/internal.cc

```diff
@@ -465,11 +465,9 @@ Status FromProto(const pb::SchemaResult& pb_result, std::string* result) {
 }
 
 Status SchemaToString(const Schema& schema, std::string* out) {
-  // TODO(wesm): Do we care about better memory efficiency here?
   ipc::DictionaryMemo unused_dict_memo;
```

Comment:
Hmm... since there seemed to be nothing concerning about memory efficiency, I thought this TODO was obsolete. What did you have in mind when writing it?